clientconn.go

```diff
@@ -96,11 +97,11 @@ func WithTimeout(d time.Duration) DialOption {
 	}
 }
 
-// WithNetwork returns a DialOption that specifies the network on which
-// the connection will be established.
-func WithNetwork(network string) DialOption {
+// WithDialer returns a DialOption that defines a function which takes an
```

Comment:
// WithDialer returns a DialOption that specifies a function to use for dialing network addresses.